File: Codigos estudiantes por lenguaje/Java/Paula Rios/matematicas/factoresPrimos.java
public class Main {
    static void primeFactors(int N){
        for (long i = 2; i*i <=N; ++i) {
            while(N%i==0){
                System.out.println(i);
                N/=i;
            }
        }
        if(N>1){
            System.out.println(N);
        }
    }
    public static void main(String[] args) {
        int n=23;
        primeFactors(42);
    }
}

File: Codigos estudiantes por lenguaje/C++/Daniel Cangrejo/Matematicas/factoresprimos.cpp
#include<bits/stdc++.h>
#include<cstdlib>
using namespace std;
typedef long long int ll;
void primeFactors(int N){
    for(ll i=2;i*i<=N;i++){
        while(N%i==0){
            cout<<i<<endl;
            N/=i;
        }
    }
    if(N>1){
        cout<<N<<endl;
    }
}
int main() {
    primeFactors(100);
}
